base/src/main/java/com/smartdevicelink/proxy/rpc/SubscribeVehicleDataResponse.java

```diff
@@ -531,4 +532,21 @@ public void setOEMCustomVehicleData(String vehicleDataName, VehicleDataResult ve
     public VehicleDataResult getOEMCustomVehicleData(String vehicleDataName){
         return (VehicleDataResult) getObject(VehicleDataResult.class, vehicleDataName);
     }
+
+    /**
+     * Sets a vehicleDataResult value for WindowStatus.
+     * @param windowStatus a VehicleDataResult value
+     */
+    public void setWindowStatus(VehicleDataResult windowStatus){
+        setParameters(KEY_WINDOW_STATUS, windowStatus);
+    }
+
+    /**
+     * Gets a VehicleDataResult value for WindowStatus.
+     * @return a VehicleDataResult object value or null.
+     * If true, means the VehicleDataResult data has been subscribed.
+     */

```

Comment:
```suggestion
    /**
     * Gets the windowStatus. If true, means the WindowStatus data has been subscribed.
     *
     * @return Boolean See WindowStatus
     * @since SmartDeviceLink 7.0.0
     */
```
Please use the [RPC generator](https://github.com/smartdevicelink/sdl_java_suite/tree/master/generator) in the sdl_java_suite project to ensure the RPC matched the spec exactly.